2020/scoring/clean_and_join_csvs.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
+
+SCORE=../../tools/scoring/score.py
+JOINSCORE=../../tools/scoring/join_results.sh
+COLORDER=../previous-best/unify_column_order.py
+SOLVERSDIVS=../registration/solvers_divisions_final.csv
+
+INC_ORIG_IN=incremental/Job_info_orig.csv
```

Comment:
Where is `INC_FIXED_IN` ?

Also, is `2020/scoring/incremental` the right directory?  I would expect it to be somewhere below `2020/result`.

Reply:
I'll add the code for the fixed stuff now that the UC is done.

Reply:
> Also, is 2020/scoring/incremental the right directory? I would expect it to be somewhere below 2020/result.

That makes sense.  I'll move the files.  Maybe I'll commit the `Job*_info.csv`s there as well.

Reply:
Fixed as suggested.